server/api/interest/__tests__/interest.spec.js

```diff
@@ -205,14 +226,38 @@ test.serial('Should update the interest state from interested to invited', async
     .set('Accept', 'application/json')
     .set('Cookie', [`idToken=${jwtData.idToken}`])
   t.is(res.status, 200)
-  t.is(res.body.status, 'invited')
+  const updateInterest = res.body
+  t.is(updateInterest.status, InterestStatus.INTERESTED)
+  t.is(updateInterest.messages.length, 2)
+  t.is(updateInterest.messages[1].name, from.nickname)
+})
+
+test.serial('Should update the interest state from interested to invited', async t => {
+  // interests 2 has a single date - so should trigger a calendar event to be attached
```

Comment:
This comment appears to have been copy and pasted and is not relevant here

Reply:
yes several times over - will remove